data/yam/agama/auto/default_sap.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
could you please rename this file to `*_no_scc.json` and create a copy with registration fields like in:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20773/files#diff-9261c14703d0439adbc6a9aaab68b65579e7d88169e6013c85afa475aadc561cR4

Reply:
I renamed the file, and created a copy with registration field, thanks.